pkgs/development/node-packages/node-packages.json

```diff
@@ -1,10 +1,15 @@
 [
   "@angular/cli"
+, "@angular/language-server"
 , "@antora/cli"
 , "@antora/site-generator-default"
 , "@bitwarden/cli"
+, "@elm-tooling/elm-language-server"
```

Comment:
dupe with https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/elm/packages/node-packages.json

Reply:
Just removed that, and added rustywind and merged with upstream.